### PR TITLE
Graphql: Add gps location of all segments to BookingInterface

### PR DESCRIPTION
**Pull request by tbergq**
We will need these coordinates to set the MMB map to correct region

### CODE REVIEW

File: apps/graphql/src/apps/booking/types/outputs/BookingInterface.js [hunk 1]
```diff
@@ -66,8 +69,42 @@ export const commonFields = {
...
+          })),
+        ];
+      }, []);
+    },
+  },
 };
 export default new GraphQLInterfaceType({
   name: 'BookingInterface',
-  fields: commonFields,
+  fields: {
```
Comment (RobinCsl) [inline apps/graphql/src/apps/booking/types/outputs/BookingInterface.js]: Is that change necessary?

Comment (tbergq) [reply apps/graphql/src/apps/booking/types/outputs/BookingInterface.js]: Absolutely not, I will undo it.

Comment (tbergq) [reply apps/graphql/src/apps/booking/types/outputs/BookingInterface.js]: 🚀

File: apps/graphql/src/apps/booking/types/outputs/BookingInterface.js [hunk 2]
```diff
@@ -66,6 +69,38 @@ export const commonFields = {
     type: GraphQLBookingType,
     description: 'OneWay, Multicity or Return',
   },
+  segmentLocations: {
+    type: GraphQLList(GraphQLCoordinate),
+    description:
+      'This will contain the gps-coordinates for all stops on the booking',
```
Comment (RobinCsl) [inline apps/graphql/src/apps/booking/types/outputs/BookingInterface.js]: In the same way we have `deprecated` flags, would it be possible to have some kind of `future feature` flag? In my opinion, a description should use present tense, but I understand how it is necessary here if there is no such `feature` flag.

Comment (tbergq) [reply apps/graphql/src/apps/booking/types/outputs/BookingInterface.js]: Yeah, you are right, I will change it to `This contains ...`

Comment (tbergq) [reply apps/graphql/src/apps/booking/types/outputs/BookingInterface.js]: 🚀
